common/source/docs/common-object-avoidance-landing-page.rst

```diff
@@ -24,20 +25,18 @@ Avoidance Features
     Simple Object Avoidance <common-simple-object-avoidance>
     Autonomous Mission Re-routing via Bendy Ruler <common-oa-bendyruler>
     Autonomous Mission Re-routing via Dijkstras <common-oa-dijkstras>
-    Cylindrical Geo-Fencing Failsafe <common-ac2_simple_geofence>
-    Polygon Geo-Fencing Failsafe <common-polygon_fence>
-[/site]
-[site wiki="plane"]
-    Geo-Fencing Failsafes <geofencing>
 [/site]
 
+
+
+
 Hardware
 ========
 
 -  :ref:`ADS-B Receiver <common-ads-b-receiver>`
-[site wiki="copter,rover"]
+
 -  :ref:`Rangefinders <common-rangefinder-landingpage>`
 -  :ref:`Fence Beacons (Non GPS navigation) <common-non-gps-navigation-landing-page>` 
```

Comment:
another pre-existing issue so not a blocker but I don't think we should call our Beacons, "Fence Beacons".  Instead we should call the Pozyx Beacons like we do on this page: https://ardupilot.org/copter/docs/common-pozyx.html.  The link should also go to this page I think.

Reply:
will do

Reply:
that should be moved...its not object avoidance per se, its non gps navigation

Reply:
should be moved to non-gps nav stuff